cover_agent/main.py

```diff
@@ -15,7 +16,10 @@ def parse_args():
         "--source-file-path", required=True, help="Path to the source file."
     )
     parser.add_argument(
-        "--test-file-path", required=True, help="Path to the output test file."
+        "--test-file-path", required=True, help="Path to the input test file."
+    )
+    parser.add_argument(
+        "--test-file-output-path", required=False, help="Path to the output test file.", default="", type=str
```

Comment:
@mrT23 Why create a new output file? I thought the idea was to append to the existing file and rollback the test if it fails?